_layouts/home.html

```diff
@@ -202,6 +202,11 @@ <h2 class="persona__heading">Plug your service into Apache OpenWhisk.</h2>
     <div class="persona__text">
       <div class="persona__columns">
         <div class="persona__column">
+          <img class="persona__benefit-image"
+               src="{{ site.github.url }}/images/illustration-contributors.svg"
+               alt="Apache OpenWhisk benefits for open-source contributors">
+        </div>
+        <div class="persona__column">
           <h2 class="persona__heading">Join a vibrant open source community</h2>
```

Comment:
I added a period after community to make this consistent with other headers in my PR.